hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/parser/record/transactionhandler/UtilRandomGenerateTransactionHandler.java

```diff
@@ -0,0 +1,47 @@
+package com.hedera.mirror.importer.parser.record.transactionhandler;
+
+import com.hedera.mirror.common.domain.entity.EntityId;
+import com.hedera.mirror.common.domain.transaction.UtilRandomGenerate;
+import com.hedera.mirror.common.domain.transaction.RecordItem;
+import com.hedera.mirror.common.domain.transaction.Transaction;
+import com.hedera.mirror.common.domain.transaction.TransactionType;
+
+import com.hedera.mirror.common.util.DomainUtils;
+import com.hedera.mirror.importer.parser.record.entity.EntityListener;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+import javax.inject.Named;
+
+@Log4j2
+@Named
+@RequiredArgsConstructor
+class UtilRandomGenerateTransactionHandler implements TransactionHandler {
```

Comment:
Should be named after the HAPI transaction not the domain:

```suggestion
class RandomGenerateTransactionHandler implements TransactionHandler {
```

Reply:
I see ok, I have renamed this and other classes/methods.